api/proquest/client.py

```diff
@@ -131,29 +132,35 @@ def missing_property(self):
         return self._missing_property
 
 
-class Book(object):
-    """POCO class containing book information."""
+class ProQuestBook(object):
+    """POCO class containing information about a ProQuest book."""
 
-    def __init__(self, link=None, content=None):
-        """Initialize a new instance of Book class.
+    def __init__(self, link=None, content=None, content_type=None):
+        """Initialize a new instance of ProQuestBook class.
 
         :param link: Book's link
         :type link: Optional[str]
 
         :param content: Book's content
-        :type content: Optional[Union[str, bytes]]
+        :type content: Optional[bytes]
+
+        :param content_type: Content type
+        :type content_type: Optional[str]
         """
         if link is not None and not is_string(link):
             raise ValueError("Argument 'link' must be a string")
         if content is not None and not isinstance(content, bytes):
             raise ValueError("Argument 'content' must be a bytes string")
+        if content_type is not None and not is_string(content_type):
+            raise ValueError("Argument 'content_type' must be a non-empty string")
```

Comment:
The logic doesn't match the message; it should be `if not content_type or not is_string(content_type)`

Reply:
Actually, all three three arguments are optional. This message should be read like this: "If present, argument 'content_type' must be a string". Can I phrase it like that or there is a better way?

Reply:
I think just "Argument x must be a string" is fine -- we don't generally do this kind of checking and I don't think it will show up in real life.

Reply:
Fixed the message accordingly